custom_components/ipx800v4/sensor.py

```diff
@@ -139,3 +148,11 @@ def __init__(
     def native_value(self) -> float:
         """Return the current value."""
         return round(self.coordinator.data[f"THL{self._id}-{self._req_type}"], 1)
+
+class XENOSensor(IpxEntity, SensorEntity):
+    """Representation of an Enocean sensor."""
+
+    @property
+    def native_value(self) -> float:
+        """Return the current value."""
+        return round(self.coordinator.data[f"ENO ANALOG{self._id - 121 + 17}"], 1)
```

Comment:
```suggestion
        return round(self.coordinator.data[f"ENO ANALOG{int(self._id) - 121 + 17}"], 1)
```
to handle case of someone put the id in string